game/src/graphic/textures/TextureHandler.java

```diff
@@ -15,17 +18,44 @@
 /**
  * A texture handler class for managing textures paths for further use.
  *
- * <p>Singleton (possibly thread-safe).
+ * <p>Singleton.
  */
 public class TextureHandler {
+    public static final String PLACEHOLDER_FILENAME = ".resource_root_please_not_modify";
     private static final TextureHandler INSTANCE = new TextureHandler();
 
     private final Map<String, Set<FileHandle>> pathMap = new LinkedHashMap<>();
 
     private TextureHandler() {
-        addAllAssets(Gdx.files.internal(Gdx.files.getLocalStoragePath()));
+        List<Path> placeholderPaths = findAllPlaceholderPaths();
+        assert placeholderPaths.size() == 2;
+        // takes the placeholder path with the longest path string...
+        if (placeholderPaths.get(0).toString().length()
+                > placeholderPaths.get(1).toString().length()) {
+            addAllAssets(new FileHandle(placeholderPaths.get(0).getParent().toFile()));
+        } else {
+            addAllAssets(new FileHandle(placeholderPaths.get(1).getParent().toFile()));
+        }
+    }
+
+    private List<Path> findAllPlaceholderPaths() {
+        try (Stream<Path> walk = Files.walk(Path.of(Gdx.files.getLocalStoragePath()), 4)) {
+            return walk.filter(
+                            x ->
+                                    Files.isRegularFile(x)
+                                            && PLACEHOLDER_FILENAME.equals(
+                                                    x.getFileName().toString()))
+                    .toList();
+        } catch (IOException ex) {
+            throw new RuntimeException(ex);
+        }
```

Comment:
was tust du hier? und warum?

da die exception in der methode sowieso nicht behandelt wird, wäre es da nicht deutlich user-freundlicher, die spezifischere original-exception hochzureichen anstatt einer allgemeinen rte? außerdem wäre dein code dann sofort lesbarer :)